salesforce/bulkwrite.go

```diff
@@ -348,17 +295,27 @@ func (c *Connector) getPartialFailureDetails(ctx context.Context, jobInfo *GetJo
 			return nil, err
 		}
 
+		fieldNames := []string{sfIdFieldName, sfErrorFieldName}
+
+		if jobInfo.ExternalIdFieldName != "" {
```

Comment:
if ` jobInfo.ExternalIdFieldName` is an empty string, should we error out? When would that happen?

Reply:
In case of delete job, it is going to be empty. for that reason in line 335, I am using switch to get correct rows according to the operation.

Reply:
But I think it will be more clear for me to add another if to check if it is delete operation